src/defaultsettings.cpp

```diff
@@ -76,11 +76,19 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("tooltip_show_delay", "400");
 
 	// Some (temporary) keys for debugging
+	#ifdef NDEBUG
```

Comment:
Debug is used in the block above here, so needs to be changed if it never exists